### ISSUE TITLE
Move our docs from Github Pages to Cloudflare Pages

### ISSUE THREAD

**Issue by infomiho** [wasp-lang/wasp#1929]
We had an issue with SSL certs for our website. The root cause turned out Github Pages cert renewal was not possible when we are using Cloudflare DNS and set the SSL to `Full (Strict)`. 

Since we are already using Cloudflare to manage our DNS and as our CDN, we should ideally move our docs to Cloudflare Pages which would simplify our docs setup.

The side benefit would be automatic publishing of our docs on pushes to the `release` branch.

**Comment by Martinsos** [comment]
Note: we might not want to automatically deploy on the push to the `release` branch, since doing it manually gives us a bit more control, making sure we keep it synced with other deployment we are doing from `release` branch. I am not saying we shouldn't do it, I am just saying I would love to thinking it through a bit more.

But we should certainly move to Cloudflare Pages!

**Comment by infomiho** [comment]
I agree, our release process includes triggering it manually when we have everything ready on the `release` branch. Publishing docs before that (and it takes around 10 mins for the release CI to pass) is a bit of a weird out-of-sync state. 

Ideally, after we release a new version, the new docs get published. Maybe even with the same CI workflow? 🤔

**Comment by infomiho** [comment]
We moved our docs to Cloudflare Pages with the wasp.sh migration. Automatic deployments are paused and we have control when we publish the docs.
